src/lib/store.ts

```diff
@@ -0,0 +1,27 @@
+import { configureStore } from "@reduxjs/toolkit";
+import counter from "../feature/counterSlice";
+import storage from "redux-persist/lib/storage";
+import { persistReducer, persistStore, PersistConfig } from "redux-persist";
+
+const persistConfig: PersistConfig<any> = {
+  key: "root",
```

Comment:
Use a better key, not the default one. This might cause conflicts with other applications